buildkite/scripts/run-test-executive-local.sh

```diff
@@ -27,7 +27,7 @@ cleanup
 TEST_NAME="$1"
 
 MINA_IMAGE="gcr.io/o1labs-192920/mina-daemon:$MINA_DOCKER_TAG-berkeley"
-ARCHIVE_IMAGE="gcr.io/o1labs-192920/mina-archive:$MINA_DOCKER_TAG"
+ARCHIVE_IMAGE="gcr.io/o1labs-192920/mina-archive:$MINA_DOCKER_TAG-berkeley"
```

Comment:
I'm trying to see in a log where this change happens. Can you point me to one? It should be in one test suffixed with `integration test local` but I cannot find any.

Reply:
Ah, wait, I'm looking at an old build from this branch.

Reply:
Here for instance: https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/3809/steps/canvas?sid=01980f90-a9b2-46cb-bc0a-8aa6a55ed10f#01980f90-a9ee-4cfb-acdc-b543084d9704/266-433